src/com/actelion/research/gwt/minimal/MoleculeQueryFeatures.java

```diff
@@ -0,0 +1,19 @@
+package com.actelion.research.gwt.minimal;
+
+import jsinterop.annotations.*;
+import com.actelion.research.chem.StereoMolecule;
+
```

Comment:
I would like to easily create a JavaScriptObject from a function but don't know how to do.